src/main/java/by/litelife/mk/pollutanttransport/model/InputData.java

```diff
@@ -1,28 +1,36 @@
 package by.litelife.mk.pollutanttransport.model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class InputData {
-    private double windSpeed;
-    private double waterFlow;
+    private List<TimeConcentrationPair> timeConcentrationPairs;
+    private String timeConcentrationPairsJson;
     private double riverSpeed;
-    private double radius;
-    private double time;
     private double lat;
     private double lng;
 
-    public double getWindSpeed() {
-        return windSpeed;
+    public InputData() {
```

Comment:
This logic must be in enpoint of controller for GET this page.

Reply:
Done.

Reply:
Delete empty constructor since you have default.